### PR TITLE
gerritstatusupdater: workarounds for broken GitHub event state

### PR DESCRIPTION
**Pull request by myitcv**
(see commit message)

### CODE REVIEW

File: internal/functions/gerritstatusupdater/gerritstatusupdater.go [hunk 1]
```diff
@@ -147,30 +148,92 @@ const (
...
+	c.DeliveryID = github.DeliveryID(r)
+
+	// Set up simple logging that closes over certain variables
+	if os.Getenv("NETLIFY") == "true" {
+		logpkg.SetFlags(0) // we get time information from Netlify
+	}
+	// We also get a uniqure prefix from Netlify, but build
+	// a context-based prefix ourselves here too.
+	logpkg.SetPrefix("")
+	log := func(format string, args ...interface{}) {
```
Comment (mvdan) [inline internal/functions/gerritstatusupdater/gerritstatusupdater.go:224]: nit: I would call this `logf`, then you don't need to rename `log` either.

File: internal/functions/gerritstatusupdater/gerritstatusupdater.go [hunk 2]
```diff
@@ -147,30 +148,92 @@ const (
...
+		result *int
+	)
+
+	c.DeliveryID = github.DeliveryID(r)
+
+	// Set up simple logging that closes over certain variables
+	if os.Getenv("NETLIFY") == "true" {
+		logpkg.SetFlags(0) // we get time information from Netlify
+	}
+	// We also get a uniqure prefix from Netlify, but build
```
Comment (mvdan) [inline internal/functions/gerritstatusupdater/gerritstatusupdater.go:221]: typo: unique

**Review by mvdan — APPROVED** [review]

File: internal/functions/gerritstatusupdater/gerritstatusupdater.go (hunk 1 above)
Comment (myitcv) [reply internal/functions/gerritstatusupdater/gerritstatusupdater.go:224]: Done.

File: internal/functions/gerritstatusupdater/gerritstatusupdater.go (hunk 2 above)
Comment (myitcv) [reply internal/functions/gerritstatusupdater/gerritstatusupdater.go:221]: Done

**Pull request merged**